### PR TITLE
Move new file to car.rb

### PR DESCRIPTION
**Pull request by s-espinosa**
@mikedao Can you take a quick look and let me know if you agree this can be merged.

* Changes name of `practice_file.rb` to `car.rb`

### CODE REVIEW

**Comment by s-espinosa** [comment]
* Additionally creates initialize method for Car

File: car.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Car
+  attr_reeder :color
```
Comment (mikedao) [inline car.rb]: Hey @s-espinosa did you actually mean `reeder` or did you want to type in `reader`?

File: car.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+  def init(color)
```
Comment (mikedao) [inline car.rb]: Hey, I think you meant `initialize`

image

**Review by mikedao — CHANGES_REQUESTED** [review]
Hey can you look at the comments and address them?

XTHXBAI

**Comment by s-espinosa** [comment]
@mikedao Good catches! Made those changes. Let me know if you see anything else. 👍 🎉 🎇 ✨

**Review by mikedao — APPROVED** [review]

**Comment by mikedao** [comment]
Looking great!

image

**Pull request merged**